src/main/java/com/scrive/pdftools/tools/utils/FileInput.java

```diff
@@ -0,0 +1,41 @@
+package com.scrive.pdftools.tools.utils;
+
+import com.itextpdf.text.pdf.codec.Base64;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.util.Objects;
+
+public class FileInput {
```

Comment:
it seems to me that this whole class can be converted into a simple static function that takes `(localFilePath, base64Content)` and returns the loaded content `byte[]` + the JSONObject unwrapping

Reply:
Yes. I have re-written this to just a static function.